frontend/src/components/Login.js

```diff
@@ -56,6 +60,14 @@ class Login extends Component {
     });
   }
 
+  renderErrors(){
+    return (
+      <p id="login-error">
+        {this.state.error}
```

Comment:
Let's move this to a generic error showing component, in its own file, so we can use that everywhere if we have an error.